Mathlib/Topology/Algebra/Module/Alternating.lean

```diff
@@ -0,0 +1,610 @@
+  [TopologicalSpace M] [TopologicalSpace N] [Module A M] [Module A N] [DistribMulAction R N]
+  [ContinuousConstSMul R N] [SMulCommClass A R N]
+
+instance [ContinuousAdd N] : DistribMulAction R (M [Λ^ι]→L[A] N) :=
+  Function.Injective.distribMulAction toMultilinearAddHom
+    toContinuousMultilinearMap_injective fun _ _ => rfl
+
+end DistribMulAction
+
+section Module
+
+variable {R A M N ι : Type _} [Semiring R] [Semiring A] [AddCommMonoid M] [AddCommMonoid N]
+  [TopologicalSpace M] [TopologicalSpace N] [ContinuousAdd N] [Module A M] [Module A N] [Module R N]
+  [ContinuousConstSMul R N] [SMulCommClass A R N]
+
+/-- The space of continuous alternating maps over an algebra over `R` is a module over `R`, for the
+pointwise addition and scalar multiplication. -/
+instance : Module R (M [Λ^ι]→L[A] N) :=
+  Function.Injective.module _ toMultilinearAddHom toContinuousMultilinearMap_injective fun _ _ =>
+    rfl
+
+/-- Linear map version of the map `to_multilinear_map` associating to a continuous alternating map
+the corresponding multilinear map. -/
+@[simps]
+def toContinuousMultilinearMapLinear :
+    M [Λ^ι]→L[A] N →ₗ[R] ContinuousMultilinearMap A (fun _ : ι => M) N where
+  toFun := toContinuousMultilinearMap
+  map_add' _ _ := rfl
+  map_smul' _ _ := rfl
+
+/-- `ContinuousAlternatingMap.pi` as a `LinearEquiv`. -/
+@[simps (config := { simpRhs := true })]
+def piLinearEquiv {ι' : Type _} {M' : ι' → Type _} [∀ i, AddCommMonoid (M' i)]
+    [∀ i, TopologicalSpace (M' i)] [∀ i, ContinuousAdd (M' i)] [∀ i, Module R (M' i)]
+    [∀ i, Module A (M' i)] [∀ i, SMulCommClass A R (M' i)] [∀ i, ContinuousConstSMul R (M' i)] :
+    (∀ i, M [Λ^ι]→L[A] M' i) ≃ₗ[R] M [Λ^ι]→L[A] ∀ i, M' i :=
+  { piEquiv with
+    map_add' := fun _ _ => rfl
+    map_smul' := fun _ _ => rfl }
+
+end Module
+
+section SmulRight
+
+variable {R A M N ι : Type _} [CommSemiring R] [AddCommMonoid M] [AddCommMonoid N] [Module R M]
+  [Module R N] [TopologicalSpace R] [TopologicalSpace M] [TopologicalSpace N] [ContinuousSMul R N]
+  (f : M [Λ^ι]→L[R] R) (z : N)
+
+/-- Given a continuous `R`-alternating map `f` taking values in `R`, `f.smul_right z` is the
+continuous alternating map sending `m` to `f m • z`. -/
+@[simps! toContinuousMultilinearMap apply]
+def smulRight : M [Λ^ι]→L[R] N :=
+  { f.toAlternatingMap.smulRight z with toContinuousMultilinearMap := f.1.smulRight z }
+
+end SmulRight
+
+section Semiring
+
+variable {R M M' N N' ι : Type _} [CommSemiring R] [AddCommMonoid M] [Module R M]
+  [TopologicalSpace M] [AddCommMonoid M'] [Module R M'] [TopologicalSpace M'] [AddCommMonoid N]
+  [Module R N] [TopologicalSpace N] [ContinuousAdd N] [ContinuousConstSMul R N] [AddCommMonoid N']
+  [Module R N'] [TopologicalSpace N'] [ContinuousAdd N'] [ContinuousConstSMul R N']
+
+/-- `ContinuousAlternatingMap.compContinuousLinearMap` as a bundled `LinearMap`. -/
+@[simps]
+def compContinuousLinearMapₗ (f : M →L[R] M') : M' [Λ^ι]→L[R] N →ₗ[R] M [Λ^ι]→L[R] N where
```

Comment:
```suggestion
def compContinuousLinearMapₗ (f : M →L[R] M') : (M' [Λ^ι]→L[R] N) →ₗ[R] (M [Λ^ι]→L[R] N) where
```
I know that the parentheses are not needed by Lean, but they can be useful to human readers as it's not everywhere the usual arrow priorities.